.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Usare IntelliSense per informazioni sui possibili attributi.
```

Comment:
Not sure if you added this file on purpose to this repository?  Don't use VScode myself.  If you think it can help others when it is here, please can you translate your comments in English?

Reply:
This file can be deleted. It was there for debugging pourposes only.